packages/TelegramClient-UI.package/TCUMessage.class/instance/maxContentWidth.st

```diff
@@ -1,4 +1,4 @@
-default values
+drawing
```

Comment:
is this not a default value?

Reply:
I wasnt sure there. `maxContentWidth` is contructed by other default values, so it did not hold an independent default value. Thus it somehow just behaves like a calculating method.

Reply:
Fair enough